server.js

```diff
@@ -93,6 +105,55 @@ app.get("/accessibilite", (_, res) => res.redirect("/#/pages/accessibilité"));
 app.get("/mentions-legales", (_, res) => res.redirect("/#/pages/mentions-légales"));
 app.get("/stats", (_, res) => res.redirect("/#/stats"));
 
+// Versions
+const versionsDir = "./versions";
+let availableVersions = [];
+
+// Loading existing versions in memory
+if (fs.existsSync(versionsDir)) {
```

Comment:
Do we really need this check? I think we always want to have this directory available, so let it crash :)

Reply:
Well, when running it locally we may not have this `versions` directory. And I love the idea that the app will still serve the current version even if no other versions are available, looks more fail safe to me.

Reply:
Should we consider that not having the `/versions` directory is a problem per se? I mean, when you clone the project and start the server you will not have the old versions locally, they are only available after running the `bin/download-github-releases.sh` script.

Reply:
Ok I now understand what you meant by running locally, where you don't necessarily care *at all* about versions.